### PR TITLE
AP-5383: Remove MTR 2A feature flag

### PR DESCRIPTION
**Pull request by agoldstone93**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-5383)

Remove feature flag for MTR-2A
Update many feature tests that were broken by the flag removal

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Review by colinbruce — APPROVED** [review]
LGTM :+1:
A couple of minor, non-blocking questions/comments

File: features/providers/mtr_accelerated/capital_disregards.feature [hunk 1]
```diff
@@ -1,47 +1,9 @@
 Feature: mandatory and discretionary capital disregards questions and flow
 # TODO: AP-5493 - This flow file can be moved to a full flow non-passported journey feature after the MTR-A feature flag is removed
```
[the remark this answers is not in the document]
Comment (agoldstone93) [reply features/providers/mtr_accelerated/capital_disregards.feature:2]: @colinbruce I was going to leave them to the tidy up ticket mentioned (AP-5493), especially as this ticket has already got quite big. Happy to do the work in here if you think it's necessary though

[the remark this answers is not in the document]
Comment (colinbruce) [reply features/providers/mtr_accelerated/capital_disregards.feature:2]: Nope - did not connect the ticket having a different number! Happy to ignore for now

File: features/providers/mtr_accelerated/main_dwelling.feature [hunk 2]
```diff
@@ -1,73 +1,9 @@
 Feature: Main dwelling place changes for MTR-Accelerated measures
 # TODO: AP-5493 - This whole file can probably be deleted after the MTR-A feature flag is removed
```
[the remark this answers is not in the document]
Comment (colinbruce) [reply features/providers/mtr_accelerated/main_dwelling.feature:2]: Nope - did not connect the ticket having a different number! Happy to ignore for now

**Pull request merged**